portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,79 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body,
+html {
+    margin: 0;
+    padding: 0;
+}
+.jumbotron {
+    align-items: center;
+    display: flex;
+    background-image: linear-gradient(rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url("/images/jumboBack.jpg");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 50vh;
+    justify-content: center;
+    position: relative;
+}
+
+.headline-text {
+    border: 5px solid white;
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 100px;
+    font-weight: bold;
+    height: 150px;
+    margin-top: 20px;
+    padding:40px;
+    position: absolute;
+    text-align: center;
+
+}
+
+#about-title {
+    font-family: 32px Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: underline;
+}
+.about {
+    font-family: 20px/3 Arial, Helvetica, sans-serif;
```

Comment:
I haven't seen the 20px/3 in a font family before. What's the purpose there?

Reply:
It was supposed to be "font" my bad. I saw it in the style guide, It's a shorthand to combine font-family, font-size, and line-height 
![Screenshot 2020-06-22 at 11 32 47 AM](https://user-images.githubusercontent.com/51325209/85323155-5dbf8b80-b47c-11ea-9ed8-d0c0bc239f1b.png)

Reply:
Ah that makes more sense.